tox.ini

```diff
@@ -193,6 +194,11 @@ deps =
     celery40,celery41,celery42: pytest>=3,<4
     pytest-mock
     opentracing
+    httpx
+    pytest-asyncio
+    aiofiles-ext
+    requests
```

Comment:
don't see these used
```suggestion
```

Reply:
aiofiles is used by the starlette FileResponse and requests is used by the starlette TestClient!